### ISSUE TITLE
Enhacement: Descriptive white space reporter error message

### ISSUE THREAD

**Issue by elit-altum** [postmanlabs/newman#2666]
1. Newman Version (can be found via `newman -v`): 5.2.2
 2. OS details (type, version, and architecture): Windows 10 x64
 3. Are you using Newman as a library, or via the CLI? CLI
 3. Did you encounter this recently, or has this bug always been there: Not a bug
 4. Expected behaviour: More descriptive error message regarding not using white spaces between reporter names.
 5. Command / script used to run Newman: ```newman run examples/sample-collection.json -r cli, json```

Though the documentation clearly mentions that while mentioning multiple reporter names the user should not use spaces after the commas, this can be looked over or forgotten while running newman. The error thrown in such a case does not point out this issue, which would enable quicker debugging. 
I would like to propose to add a line in the error statement to consider removing the white space before reporter names.

**BEFORE**
![image](https://user-images.githubusercontent.com/41413622/111199831-1ae3ac00-85e7-11eb-94b1-b59b8c9194fd.png)

**AFTER**
![image](https://user-images.githubusercontent.com/41413622/111199747-02739180-85e7-11eb-94ab-dd6bc36a03bf.png)

I have a PR ready for this, I would like to discuss the exact wordings of the error messages before publishing the PR 🚀

**Comment by elit-altum** [comment]
@shamasis Please have a look!

**Comment by shamasis** [comment]
Yeah definitely - this needs fixing. But is there a way we can completely get rid of this error scenario?

If it is a parsing error, can we not prune blank reporter options before passing the options further down the line in code?

I would rather attempt at fixing the root cause than suppress the symptom. Only and only if the root cause resolution is a massive endeavour is when I would go the message route for this case.

**Comment by elit-altum** [comment]
Absolutely @shamasis, I tried to find the location where the parsing is happening but I wasn't able to locate it, is it a ```utils``` function?
Could you please guide me in the right direction?

**Comment by shamasis** [comment]
I'm afraid that would be very difficult for me at the moment.

Try putting a breakpoint on the error and observe the stack trace. 

Basic CLI parsing is done using an external library and then a bit of organising is done on top of it. Most of the code around that is in the cli bin js file.

**Comment by johnmusk** [comment]
@shamasis Why this issue is still open I think @elit-altum solve the issue or not

**Comment by shamasis** [comment]
Yeah. Waiting for @codenirvana to review once. He's a bit stuck on some other releases. Let me see if I can bump it up.
